py-utils/src/utils/msg_bus/message_bus/admin.py

```diff
@@ -0,0 +1,17 @@
+
+from bus.client import BusClient
```

Comment:
What is the purpose of this class ? Please remove or move to separate patch if not part of phase 1.

Reply:
BusClient is a abstract base class. Producer , Consumer are types of BusClient